packages/list_bloc/lib/src/cubits/list_cubit.dart

```diff
@@ -20,5 +20,68 @@ class ListCubit<T, F> extends DataCubit<List<T>, F> {
       rethrow;
     }
   }
-// TODO(alexis): implement remove, add, replace etc
+
+  void add(T item) {
+    final data = state.data;
+    emit(
```

Comment:
I do not think we need to emit loading state while not calling the api (for all methods).

What is the thinking behind it?

Reply:
I did not include it at first. But then thought maybe you would want to emit loading in these methods.
Should i remove them ?